### DISCUSSION TITLE
Week 4- Thoughts on Activities and Project Ideas

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#101] [General]
Liked: 
I liked the CSS Diner again because it was familiar and it helped me to understand the code a bit better because of the interactivity of it. I like being able to see the results of my code and interact with it. 
Disliked: 
I wasn't a huge fan of the registration form exercise. It was very difficult for me and I had to look up a lot of tutorials online because of the lacking hints/tutorials from the exercise. It would be nice to have more examples within the exercise. In the end, though, I felt like I had learned a lot. 

Feedback on some games: 
Elevator: 
This was fun! Again I liked the animations associated with the code so that I could visualize what I was actually telling the function to do. 
Code Combat:
I have previously played this game and I loved it because it provides a lot of guides and fun tutorials on how to write specific actions for your hero. 

Updated Project Ideas: 

- Portfolio 
This would be a website to showcase past projects and assignments that I have completed. 
- Online game
This could be fun, I don't know if I will make this a learning game or just for fun. 
- Recruitment Website
This would be a recruitment website for the university for prospective students so that they can access helpful tools and resources. 
- Blog 
A fun hub of interaction that I can code out to be specific for different activities that I have done.

**Comment by gabebott** [comment]
I agree! The animations in the games help to understand the intended code so much more!

**Comment by audreyhki** [comment]
@CJ3315, I also thought to create a game for a project idea and this week really helped solidify that it might be the most enjoyable project to complete. Even the FreeCodeCamp tutorial projects feel sort of like an opportunity to do an enjoyable activity, more active than passively reading and re-writing information.
